test/Interop/Cxx/symbolic-imports/print-symbolic-module-interface.swift

```diff
@@ -98,3 +111,10 @@ public:
 // CHECK-NEXT:     typealias Y = Any
 // CHECK-NEXT:   }
 // CHECK-NEXT: }
+// CHECK: class MyImmortal {
+// CHECK-NEXT:   func foo()
+// CHECK-NEXT: }
+// CHECK-NEXT: struct NonCopyable {
```

Comment:
Shouldn't this need a conformance to `~Copyable`?

Reply:
`swift-ide-test -print-module` doesn't explicitly print this conformance in the module interface